node/pkg/fetcher/fetcher.go

```diff
@@ -225,34 +226,94 @@ func (f *Fetcher) fetch(adapter AdapterDetail) ([]float64, error) {
 	adapterFeeds := adapter.Feeds
 
 	data := []float64{}
+	dataChan := make(chan float64)
+	errChan := make(chan error)
+
+	var wg sync.WaitGroup
+	wg.Add(len(adapterFeeds))
```

Comment:
@nick-bisonai Very minor. All our feeds are adapter feeds, it would be fine to call it just `feeds` than `adapterFeeds`.